src/server/admin/teams/controllers/save/team-details.js

```diff
@@ -13,11 +13,12 @@ const teamDetailsController = {
     const payload = request?.payload
     const redirectLocation = payload?.redirectLocation
 
-    const name = payload?.name || undefined
-    const description = payload?.description || undefined
-    const serviceCode = payload?.serviceCode || undefined
-    const alertEmailAddresses =
-      payload?.alertEmailAddresses?.split(/\s*,\s*/) || undefined
+    const name = payload.name
+    const description = payload.description || undefined
```

Comment:
would not `const description = payload?.description` achieve the same? Nitpicking though

Reply:
No because optional chaining works with `undefined` or `null` - these values when nothing has been sent are empty strings `""` - so optional chaining does nothing - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining